<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame_error">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
background-color: rgba(0, 0, 0, 0.5);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="we" native="true">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>468</y>
      <width>552</width>
      <height>345</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Images/切图/弹窗BG.png);
background: transparent;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>552</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">image: 0;
background: 0;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>用户名或密码错误，请重新输入！</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_yes">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>230</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{image: 0; border-image: url(:/Images/切图/确--认-拷贝.png);}
QPushButton:pressed{image: 0; border-image: url(:/Images/切图/确--认-拷贝-2.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exit">
     <property name="geometry">
      <rect>
       <x>485</x>
       <y>40</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{image: 0; background: 0; border-image: url(:/Images/切图/关--闭-拷贝.png);}
QPushButton:pressed{image: 0; background: 0; border-image: url(:/Images/切图/关--闭-拷贝-按下.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_input">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1280</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>793</y>
      <width>428</width>
      <height>72</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Images/切图/圆角矩形-1.png);</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>80</width>
       <height>72</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/切图/mima-(2).png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>350</width>
       <height>72</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">image: 0;
color: rgb(255, 255, 255);
background:transparent;
border-style:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入您的密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>983</y>
      <width>222</width>
      <height>72</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
background-color: rgb(13, 225, 255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Images/切图/BG2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>frame_input</zorder>
  <zorder>frame_error</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
